jwst/regtest/test_miri_lrs_slit_spec2.py

```diff
@@ -33,7 +36,7 @@ def run_pipeline(rtdata_module):
 @pytest.mark.bigdata
 @pytest.mark.parametrize("suffix", [
     "assign_wcs", "combinedbackground", "bsub", "srctype", "flat_field", "pathloss",
-    "cal", "s2d", "x1d"])
+    "cal", "photom", "pixel_replace", "s2d", "x1d"])
```

Comment:
don't need photom if you've got cal in the list already

Reply:
ok